namespaces/cloud-platform-live-0.k8s.integration.dsd.io/omic-allocation-prototype/00-namespace.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: omic-allocation-prototype
```

Comment:
Can you please include the labels and annotations as in the [template](https://github.com/ministryofjustice/cloud-platform-environments/blob/master/namespace-resources/00-namespace.yaml)?